Update Experiment docs for default deployment (#203)
amplitude/amplitude-dev-center: docs/experiment/guides/getting-started/index.md

This guide walks through getting started with Amplitude Experiment. For the best experience, read the guide in this order. You can use the navigation at the bottom of each page to move to the next section.

> 1. [Create a flag](./create-a-flag.md), and add a [deployment](../../general/data-model.md#deployments). Flags alone are used for standard feature flagging without user analysis. Experiments are used for feature experimentation on users. Learn more about [flags and experiments](../../general/data-model/#flags-and-experiments).

Reviewer: ```suggestion
1. [Create a flag](./create-a-flag.md), and add a [deployment](../../general/data-model.md#deployments). Use flags by themselves for standard feature flagging without user analysis. Use experiments for feature experimentation on users. Learn more about [flags and experiments](../../general/data-model/#flags-and-experiments).
```

Reviewer 2: ```suggestion
1. [Create a flag](./create-a-flag.md), and add a [deployment](../../general/data-model.md#deployments). Use flags for standard feature flagging without user analysis. Use experiments for feature experimentation on users. Learn more about [flags and experiments](../../general/data-model/#flags-and-experiments).
```
